### DISCUSSION TITLE
v2.9.4

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4970] [分享 / Share]
✨新增
- `屏蔽黑名单up主` 去除了登录校验. (#4917, PR #4926 by [snowraincloud](https://github.com/snowraincloud))
- `图片批量导出` 适配新型的图片动态. (#4830)
- 更新评论区的 `夜间模式`. (#4931)

🐛修复
- 修复 StreamSaver 和 ffmpeg WASM 的 SRI 校验问题. (#4913, #4864)
- 修复下载合集时 BV 号命名重复. (#4818)
- 修复 `删除广告` 去除首页广告时没有遵循 `占位文本` 选项. (#4836)
- 修复 `自定义顶栏` 历史面板中的多余 title 提示. (#4962)
- 修复下载弹幕时播放器设置没有正确读取. (#4824)

🗑️废弃
- `自定义顶栏` 删除 `相簿` 入口.

☕开发者相关
- 修复 DevClient 无法重载 Shadow DOM 样式.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.9.4'>v2.9.4</a>.</em>

**Comment by kethoven** [comment]
传统连播功能 好像失效了。
好麻烦
